Integrations/src/test/java/io/deephaven/integrations/numpy/Java2NumpyCopyTest.java

```diff
@@ -1,782 +0,0 @@
-package io.deephaven.integrations.numpy;
```

Comment:
This PR deleted this test suite!  Looks like this got botched in backing out the changes.

Reply:
This was intentional; I got rid of it because Java2NumpyCompyTest was the only thing that existed in there and it is the failing test that I'm working on elsewhere. Else I suppose I should have just left the broken test there?